plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -117,31 +117,32 @@ func getResourceRequest(pod *api.Pod) resourceRequest {
 	return result
 }
 
-func CheckPodsExceedingFreeResources(pods []*api.Pod, capacity api.ResourceList) (fitting []*api.Pod, notFittingCPU, notFittingMemory []*api.Pod) {
+func CheckPodsExceedingFreeResources(pods []*api.Pod, capacity api.ResourceList) (bool, bool) {
```

Comment:
If all the users of this method are passing only a single pod, then I'd prefer updating the signature to take just one pod as input. Otherwise, I prefer the previous signature.